web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
90% of the times that I go to a Prometheus server, I want to explore/graph data, so I agree that /graph would be a better entry point. I'm not really worried about new users since the top menu is clearly visible. Other time series databases (e.g. InfluxDB) also usually have the querying page (or closest equivalent) as their default page. It might not be the perfect end outcome yet, but a better default than /status for sure.